streams/sy.m3u

```diff
@@ -31,3 +31,7 @@ http://vod.ortas.sy:1935/oLive/snews05042022/playlist.m3u8
 http://vod.ortas.sy:1935/oLive/sat05042022/playlist.m3u8
 #EXTINF:-1 tvg-id="AlalamNewsChannelSyria.sy",Alalam News Channel Syria (576p) 
 https://live2.alalam.ir/alalam.m3u8
+#EXTINF:-1 tvg-id="AlyaumChannel.sy",Alyaum Channel (720p) 
+https://ikomg1.s.llnwi.net/alyaum-tv/alyaum-tv.m3u8
```

Comment:
```suggestion
```
It's already in https://github.com/iptv-org/iptv/blob/2c5231185b6f7a2d48f0d376e0de61b1c02d1ad2/streams/ae.m3u#L47 . Additionally, "AlyaumChannel.sy" is not a valid tag